twist_mux:
  ros__parameters:
    topics:
      nav2:
        topic : cmd_vel
        timeout : 0.5
        priority : 10
      # joystick:
      #   topic : cmd_vel_joy
      #   timeout : 0.5
      #   priority : 100

      e_stop:
        topic : e_stop
        timeout : 0.5
        priority : 255
